SB2018/2018-01-12.md

```diff
@@ -0,0 +1,34 @@
+---
+to: All teams
+subject: SourceBots Mentoring and Tech Days
+---
+
+Hi all,
+
+We hope you're progressing well on your robot!
+Some important things to announce so keep reading:
```

Comment:
Maybe "We have some .." would read better?

Reply:
Or, as I've just seen @RealOrangeOne commented, just dropping this line would be fine.